### PR TITLE
Create 070.py

### PR DESCRIPTION
**Pull request by lennartwesdijk**

### CODE REVIEW

File: plugins/070.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from will.plugin import WillPlugin
+from will.decorators import respond_to
+
+class NulZeventigPlugin(WillPlugin):
+    @respond_to("070")
+    def nul_zeventig_plugin(self, message, argv):
+		if len(argv) > 1:
+			if len(sys.argv[1]) > 0:
+				r = requests.get("http://looq.nl/070/index.php?text="+"{}".format("+".join(sys.argv)))
```
Comment (pubkraal) [inline plugins/070.py]: This is pretty bad... use requests like a man and provide the querystring key/values using a dict.

Comment (pubkraal) [inline plugins/070.py]: or use it like an independent woman in the exact same way. we're PC here.

**Comment by joshp123** [comment]
pfffffff

**Comment by joshp123** [comment]
i still dont get this one

**Comment by Marfru** [comment]
not gonna happen

**Comment by githuib** [comment]
@joshp123 You should read some "Haagse Harry" to advance your dutch skills.

**Comment by joshp123** [comment]
please link me and watch as i get very confused

**Comment by githuib** [comment]
https://www.youtube.com/watch?v=ZxtuNorwYXI

**Pull request closed without merging**